src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/UpgradeAssistantHostExtensions.cs

```diff
@@ -75,10 +75,11 @@ public static IHostBuilder UseUpgradeAssistant<TApp>(this IHostBuilder host, IUp
 
                     services.TryAddTransient<IUserInput, ConsoleCollectUserInput>();
 
-                    services.AddAnalysis(options =>
+                    services.ConfigureOutputOptions(options =>
                     {
                         options.Format = upgradeOptions.Format ?? context.Configuration["Analysis:DefaultFormat"];
```

Comment:
Should we change the config section this is in? Perhaps `Output`? e.g.
```suggestion
                        options.Format = upgradeOptions.Format ?? context.Configuration["Output:DefaultFormat"];
```